python/guide/packages/gluon/trainer.md

```diff
@@ -97,6 +116,29 @@ trainer.set_learning_rate(0.1)
 trainer.learning_rate
 ```
 
-In addition, multiple pre-defined learning rate scheduling methods are
-implemented in the `mxnet.lr_scheduler` module. Please refer to [its
-tutorial](../lr_scheduler.md).
+
+
+In addition, there are multiple pre-defined learning rate scheduling methods that are already implemented in the [`mxnet.lr_scheduler`](http://beta.mxnet.io/api/gluon-related/mxnet.lr_scheduler.html) module. The learning rate schedulers can be incorporated into your trainer by passing them in as an `optimizer_param` entry. Please refer to [the lr scheduler guide](../lr_scheduler.md) to learn more.
```

Comment:
[the lr scheduler guide] -> the [LR scheduler guide]
lr -> LR

Reply:
done